Jellyfin.Api/Models/MediaSegmentsDtos/MediaSegmentDto.cs

```diff
@@ -0,0 +1,99 @@
+using System;
+using Jellyfin.Data.Entities.MediaSegment;
+using Jellyfin.Data.Enums.MediaSegmentAction;
+using Jellyfin.Data.Enums.MediaSegmentType;
+
+namespace Jellyfin.Api.Models.MediaSegmentsDtos;
+
+/// <summary>
+/// Media Segment dto.
+/// </summary>
+public class MediaSegmentDto
+{
+    /// <summary>
+    /// Gets or sets the start position in Ticks.
+    /// </summary>
+    /// <value>The start position.</value>
+    public long StartTicks { get; set; }
+
+    /// <summary>
+    /// Gets or sets the end position in Ticks.
+    /// </summary>
+    /// <value>The end position.</value>
+    public long EndTicks { get; set; }
+
+    /// <summary>
+    /// Gets or sets the Type.
+    /// </summary>
+    /// <value>The media segment type.</value>
+    public MediaSegmentType Type { get; set; }
+
+    /// <summary>
+    /// Gets or sets the TypeIndex which relates to the type.
+    /// </summary>
+    /// <value>The type index.</value>
+    public int TypeIndex { get; set; }
+
+    /// <summary>
+    /// Gets or sets the associated MediaSourceId.
+    /// </summary>
+    /// <value>The id.</value>
+    public Guid ItemId { get; set; }
+
+    /// <summary>
+    /// Gets or sets the associated MediaStreamIndex.
+    /// </summary>
+    /// <value>The id.</value>
+    public int StreamIndex { get; set; }
+
+    /// <summary>
+    /// Gets or sets the creator recommended action. Can be overwritten with user defined action.
+    /// </summary>
+    /// <value>The media segment action.</value>
+    public MediaSegmentAction Action { get; set; }
```

Comment:
```suggestion
    public MediaSegmentAction RecommendedAction { get; set; }
```

Since we've changed this from "do this action" to "client may use this action"